CONTRIBUTING.md

```diff
@@ -857,7 +857,7 @@ build_with_asan()
   LDSHARED="clang --shared" \
   LDFLAGS="-stdlib=libstdc++" \
   CFLAGS="-fsanitize=address -fno-sanitize-recover=all -shared-libasan -pthread" \
-  CXX_FLAGS="-pthread" \
+  CXXFLAGS="-fsanitize=address -fno-sanitize-recover=all -shared-libasan -pthread" \
```

Comment:
This shouldn't be necessary, but instead we should pass `USE_ASAN=1` environment variable.